src/seaport.ts

```diff
@@ -867,12 +870,40 @@ export class OpenSeaPort {
     const { buy, sell } = assignOrdersToSides(order, matchingOrder)
 
     const metadata = this._getMetadata(order, referrerAddress)
-    const transactionHash = await this._atomicMatch({ buy, sell, accountAddress, metadata })
+
+    let transactionHash: string = ''
+
+    // We use a fee wrapper to handle fee distribution instead of Wyvern
+    if (isFeeWrapperFlow({ buy, sell }, this._networkName)) {
+      const vrs = await this._authorizeOrder(matchingOrder)
+      const properlySignedMatchingOrder = { ...matchingOrder, vrs }
+      transactionHash = await this._atomicMatch({
+        buy:
+          properlySignedMatchingOrder.side === OrderSide.Buy
+            ? properlySignedMatchingOrder
+            : buy,
+        sell:
+          properlySignedMatchingOrder.side === OrderSide.Sell
+            ? properlySignedMatchingOrder
+            : sell,
+        accountAddress,
+        metadata,
```

Comment:
how come you don't pass in `makerOrder` here?

Reply:
i dont think you need it in line  898 since `if (useFeeWrapper && makerOrder?.hash) {`
will only ever be false with useFeeWrapper = false

Reply:
ah yeah, mistake